### PR TITLE
Remove scream refs

### PR DESCRIPTION
**Pull request by bartgol**
There should be no occurrence of the word "scream" in any file name/content.

### CODE REVIEW

File: bkp/bkp_ekat_assert.hpp [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+#include <sstream>
+#include <exception>
+#include <assert.h>
+#include <stdexcept>  // For std::logic_error
+
+#include <cfenv>  // For FPE environment
+
+/*
+ * Asserts and error checking for Scream.
```
Comment (jeff-cohere) [inline bkp/bkp_ekat_assert.hpp:12]: These are comments, so not a big deal, but the word "scream" does appear here. :-)

File: bkp/bkp_ekat_assert.hpp [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+#endif
+
+#define ekat_require(condition)                 impl_throw(condition, "",  std::logic_error)
+#define ekat_require_msg(condition, msg)        impl_throw(condition, msg, std::logic_error)
+
+#define ekat_kernel_require(condition)          impl_kernel_throw(condition, "")
+#define ekat_kernel_require_msg(condition, msg) impl_kernel_throw(condition, msg)
+
+#define ekat_error_msg(msg)                     scream_require_msg(false, msg)
```
Comment (jeff-cohere) [inline bkp/bkp_ekat_assert.hpp:66]: This shouldn't dispatch to scream_require_msg, should it?

File: bkp/bkp_ekat_assert.hpp [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+  {
+    std::fprintf(stderr, "Assertion failure: %s\n", sz);
+  }
+};
+}
+#define CONSTEXPR_ASSERT(CHECK) \
+    ( (CHECK) || (throw impl::assert_failure(#CHECK), false) )
+#endif
+
+namespace scream {
```
Comment (jeff-cohere) [inline bkp/bkp_ekat_assert.hpp:93]: ekat?

File: bkp/bkp_ekat_assert.hpp [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+ *          FPEs for a few lines (perhaps you do some dirty trick
+ *          that is correct, but would throw an fpe), don't forget
+ *          to re-enable them after you're done.
+ */
+
+#ifdef EKAT_FPE
+// #pragma STDC FENV_ACCESS ON
+#endif
+
+static unsigned int constexpr scream_default_fpes =
```
Comment (jeff-cohere) [inline bkp/bkp_ekat_assert.hpp:122]: Here and below there are still a few screams.

File: bkp/bkp_ekat_assert.hpp (hunk 1 above)
Comment (bartgol) [reply bkp/bkp_ekat_assert.hpp:12]: Ah, the person who invented uppercase...

File: bkp/bkp_ekat_assert.hpp (hunk 2 above)
Comment (bartgol) [reply bkp/bkp_ekat_assert.hpp:66]: why isn't my git grep showing these? Thanks!

Comment (jeff-cohere) [reply bkp/bkp_ekat_assert.hpp:66]: You're just trying to make me feel valued as a review!

**Comment by bartgol** [comment]
Ah, I see what happened: I accidentally staged a bkp file I was creating for future work. I purged it, and [force pushed](https://images.thoughtbot.com/git-push-force-with-lease/XEDULrg2QsnzIGytOMfh_XFQLB.jpg).

**Pull request merged**